Modules/posixmodule.c

```diff
@@ -2394,9 +2395,8 @@ fill_time(PyObject *module, PyObject *v, int s_index, int f_index, int ns_index,
         goto exit;
 
     float_s = PyFloat_FromDouble(sec + 1e-9*nsec);
-    if (!float_s) {
+    if (!float_s)
```

Comment:
I think that style-only change here should be reverted. Diff is more important than braces position.

Reply:
Done. I thought that I've added this one :-)